source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -129,6 +129,9 @@ let parse_models_and_queries_from_sources
     ~source_sink_filter
     ~callables
     ~stubs
+    ~python_major_version
+    ~python_minor_version
+    ~python_micro_version
```

Comment:
Could we create a module for this in `modelParser.ml` and `modelParser.mli` to be less verbose? Maybe something like

```
module PythonVersion = struct
  type t = {
    major_version: int;
    minor_version: int;
    micro_version: int;
  }
end
```